### PR TITLE
chore: migrate to firestore

### LINKED ISSUE #410: Replace PostgresSQL with FireStore
The current implementation uses PostgresSQL to store the data.

This solution requires an instance on GCP to manage a minimal quantity of data.

Replace the persistence layer with FireStore to allow deployment without persistent instances.

### PR DESCRIPTION
**Pull request by RomuloVitoi**
Closes #410

### CODE REVIEW

File: plugins/verifyRequest.js [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
 import { createVerificationSignature } from '../helpers/crypto.js'
 
-export default async function verifyRequest(pluginOptions, req) {
-  const { clientid } = req.headers
+export default async function verifyRequest(req) {
   const timestamp = req.headers['x-zm-request-timestamp']
   const signature = createVerificationSignature(timestamp, req.body)
 
-  if (
-    clientid !== pluginOptions.clientId ||
```
Comment (brainrepo) [inline plugins/verifyRequest.js:10]: Maybe I am missing some context, but why are you removing the client.id check?

Comment (RomuloVitoi) [reply plugins/verifyRequest.js:10]: They stopped sending the client id header. You can check in their documentation that in the Node.js example they do not check the client id: https://developers.zoom.us/docs/api/rest/webhook-reference/#verify-with-zooms-header

**Comment by brainrepo** [comment]
Not tested, but code-wise LGTM.

**Review by brainrepo — APPROVED** [review]

File: plugins/verifyRequest.js (hunk 1 above)
Comment (simoneb) [reply plugins/verifyRequest.js:10]: @RomuloVitoi if they're not sending it anymore, can you explain why this check kept working?

**Review by simoneb — APPROVED** [review]
Looked at the code and the changes aren't easy to follow, plus testing it locally is more effort than it's worth, so I'll approve and merge and if something breaks we'll get back to this, but overall this looks good. Thanks for all this work!

**Pull request merged**